src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,58 @@
+package duke;
+
+import duke.task.*;
+
+import java.io.*;
+import java.nio.file.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    protected File file;
+
+    public Storage(String fileInput) throws DukeException{
```

Comment:
Probably you have missed this out, but maybe you would want to have a space between DukeException and the bracket.
```suggestion
   public Storage(String fileInput) throws DukeException {
```